Informing a regex syntax should include required slash separators (#35234)
github/docs: content/search-github/github-code-search/understanding-github-code-search-syntax.md

## Case sensitivity

> By default, code search is case-insensitive. Searching for `True` will include results for _uppercase_ `TRUE` and _lowercase_ `true`. You can do case-sensitive searches by using a regular expression with case insensitivity turned off, for example `/(?-i)True/`.

Reviewer: ````suggestion
By default, code search is case-insensitive, and results will include both uppercase and lowercase results. You can do case-sensitive searches by using a regular expression with case insensitivity turned off. For example, to search for the string "True",  you would use:

```text
/(?-i)True/
```
````

## Case sensitivity

> By default, code search is case-insensitive, and results will include both uppercase and lowercase results. You can do case-sensitive searches by using a regular expression with case insensitivity turned off. For example, to search for the string "True",  you would use:

Reviewer: ```suggestion
By default, code search is case-insensitive, and results will include both uppercase and lowercase results. You can do case-sensitive searches by using a regular expression with case insensitivity turned off. For example, to search for the string "True", you would use:
```

Just removing a bit of extra whitespace introduced by mistake
